### ISSUE TITLE
Checkbox not shown in dark mode

### ISSUE THREAD

**Issue by jeangagne555** [Alexhuszagh/BreezeStyleSheets#25]
When using the dark stylesheet, checkboxes are not displayed properly. Only the test is visible. See attached images comparing the default Qt stylesheet versus the dark stylesheet. 

Qt 6.2.2  with C++ in Visual Studio 2022 with Qt Visual Studio Tools version 2.8.1 (rev.6)

![image](https://user-images.githubusercontent.com/45470913/160949608-d50c2b8e-101d-4410-a2f4-cbba9fd44ff2.png)

![image](https://user-images.githubusercontent.com/45470913/160949648-52f097b6-308b-4579-8fff-d264fc4ced9d.png)

**Comment by Alexhuszagh** [comment]
Seems like the test suite in the [test suite](https://github.com/Alexhuszagh/BreezeStyleSheets/blob/4a04f66a0b7bdf6cb949163310dec3ac92cdf14f/test/ui.py) isn't catching this issue.

Going to verify on Windows (I believe it works for me on Linux with KDE), and then try to recreate your settings UI to see if I can dissect the issue. If you have time, can you try running the following command and pasting the output:

**PyQt5**

```bash
pyrcc5 dist/breeze.qrc -o breeze_resources.py
python3 test/ui.py --widget checkbox --stylesheet dark
```

**PyQt6**

```bash
python3 test/ui.py --widget checkbox --stylesheet dark --pyqt6
```

![Example Checkbox](https://user-images.githubusercontent.com/9440903/165320445-9887dd58-304f-4e8d-bb92-3e0760f0dc85.png)

**Comment by Alexhuszagh** [comment]
Here's the same image on Windows 10, using PyQt5, so I believe the issue is likely UI-specific. I'll try to recreate the above UI, but sample code (in C++ or PyQt) would be greatly appreciated.
![2022-04-26 09_37_09-Window](https://user-images.githubusercontent.com/9440903/165325322-6266ddb8-379a-495a-a33d-e95abe588334.png)

**Comment by Alexhuszagh** [comment]
I currently cannot reproduce this, so a more detailed example would be necessary to fix this issue.

### Settings Code

https://github.com/Alexhuszagh/BreezeStyleSheets/blob/040d6d260fec7b59c38341ad537da488558ac7e9/test/ui.py#L948-L1058

### Results

**Dark Style**

![Screenshot_20220426_112214](https://user-images.githubusercontent.com/9440903/165347429-8ec3ec91-3239-4dc4-8e5b-3c90225d4ab7.png)

**Light Style**

![Screenshot_20220426_112230](https://user-images.githubusercontent.com/9440903/165347466-6c11b495-b364-4446-8635-d5a8b6de77a9.png)

**Native Style**

![Screenshot_20220426_112244](https://user-images.githubusercontent.com/9440903/165347489-b686fcd8-e338-4698-895e-6855442702b6.png)

### Summary

As you can see, for my limited example, all of them have the desired behavior. I even added hooks to ensure the `QFontDialog` and `QFileDialog` events were properly getting fired and the values stored.

**Comment by jeangagne555** [comment]
As soon as I have the time I will try to recreate the issue with a simple example application.

**Comment by Alexhuszagh** [comment]
Any progress on this?

**Comment by Alexhuszagh** [comment]
If you can create a reproducible test case, I'll be glad to reopen this issue.

**Comment by starseeker** [comment]
I'm seeing similar behavior, but oddly it seems to be specific to the Qt distributed with Ubuntu.

**Comment by starseeker** [comment]
Version 22.10 (https://ubuntu.com/download/desktop)

**Comment by starseeker** [comment]
I doubt it's useful for test cases purposes, but just so you'll know what I'm doing (and whether anything I'm doing wring might be at issue)  my application is setting the theme at startup as follows:
https://github.com/BRL-CAD/brlcad/blob/main/src/qged/main.cpp#L237

I"m bundling it into the build here:
https://github.com/BRL-CAD/brlcad/blob/main/src/qged/CMakeLists.txt#L65

I'm using only the dark theme, so the files in https://github.com/BRL-CAD/brlcad/tree/main/src/qged/theme are a subset.

**Comment by starseeker** [comment]
For the most start difference, the upper right corner usually looks something like this:
![image](https://user-images.githubusercontent.com/238416/206086535-7375b732-34d0-4073-91cd-8bf6a3564b97.png)

**Comment by starseeker** [comment]
Currently I'm seeing this: 
![image](https://user-images.githubusercontent.com/238416/206086630-f09a27af-f9ce-4125-9b21-4398ea0945c0.png)

**Comment by starseeker** [comment]
There we go - looks like it is something about Ubuntu's version.  Building 6.4.2 Qt from source, I'm getting this:  
![image](https://user-images.githubusercontent.com/238416/206095619-7e9528ae-9999-439a-bc7c-7c1493155f95.png)

**Comment by starseeker** [comment]
Just curious - were you able to reproduce the issue with the Ubuntu Qt?

**Comment by starseeker** [comment]
I should follow up on my above screenshots - the icons missing were due to my system not having the qtsvg package present.  (I didn't realize svg was separate from the base install initially.)  I'm not sure if that's also related to the checkbox drawing?

**Comment by Alexhuszagh** [comment]
I'll add this to the documentation.
